src/invoker.js

```diff
@@ -5,11 +5,10 @@ var toString = require('./toString');
 
 
 /**
- * Turns a named method with a specified arity into a function that can be
- * called directly supplied with arguments and a target object.
- *
- * The returned function is curried and accepts `arity + 1` parameters where
- * the final parameter is the target object.
+ * Given an `arity` (Number) and a `name` (String) the `invoker()` function 
```

Comment:
i think you can omit the parens after `invoker`. Otherwise, this looks good to me.